aces/Loris
# [Configuration] Adding mnc2bids converter configuration settings

Includes an SQL patch and changes to `SQL/0000-00-03-ConfigTables.sql` to add configuration settings to the Configuration module for the MINC to BIDS converter script
**To be discussed:** Descriptions and labels for each setting and the initial values for each setting

Below is a view of the new addition to the Configuration module:
<img width="1307" alt="Screen Shot 2021-06-18 at 10 22 22 AM" src="https://user-images.githubusercontent.com/35467361/122577511-df5ee080-d020-11eb-9772-8cc205a80b91.png">


#### Testing instructions 

1. Source the provided SQL patch
2. Check that the `MINC to BIDS Converter` settings appear in the Configuration module in your frontend
3. Check that submitting values for the related settings correctly updates the value in the frontend and backend

* Resolves #7477
